charts/spiffe-helper-csi-driver/templates/_spire-lib.tpl

```diff
@@ -0,0 +1,26 @@
+{{- define "spire-lib.registry" }}
+{{- if ne (len (dig "spire" "image" "registry" "" .global)) 0 }}
```

Comment:
Please use the (((x).y).z) pattern of protecting against missing elements instead of dig.

Reply:
Anything _spire-lib.tpl related was just copy/pasted here since at the time the pr was written, spire-lib was not broken out to its own chart. This is fixed now in trunk, and once the pr is rebased, all traces of the _spire-lib.tpl file will go away in this pr.

Its probably worth noting these changes to be made to charts/spire/charts/spire-lib in a seperate issue so they can be made to the main file still containing the recommended changes. Could you please file an issue?